### PR TITLE
update syntax-highlight from v1.0.b31 to v2.0.3

### PR DESCRIPTION
**Pull request by AdamXweb**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask {{cask_file}}` is error-free.
- [X] `brew style --fix {{cask_file}}` reports no offenses.

### CODE REVIEW

**Comment by i0ntempest** [comment]
2.0.1 is out now

**Comment by AdamXweb** [comment]
Updated version to latest post-release fix V2.0.1

**Comment by i0ntempest** [comment]
And just now 2.0.2 is out… looks like he’s pushing bug fixes fast.

**Comment by cho-m** [comment]
CI failure seems that using `github_prerelease_allowlist.json` results in failure when version is set to an official release

If the project is no longer in alpha/beta (pre-release), you may want to remove it from allowlist:
https://github.com/Homebrew/homebrew-cask/blob/1e7fe1c9e7016f2ee61ddcf7de96a59b21bdcd52/audit_exceptions/github_prerelease_allowlist.json#L31

**Comment by AdamXweb** [comment]
Updated version to latest fix v2.0.2

**Comment by AdamXweb** [comment]
Removed syntax-highlight from prerelease list due to proper release

File: Casks/syntax-highlight.rb [hunk 1]
```diff
@@ -1,13 +1,27 @@
+# typed: false
+# frozen_string_literal: true
+
```
Comment (SMillerDev) [inline Casks/syntax-highlight.rb:3]: This shouldn't be added

**Comment by i0ntempest** [comment]
That frozen literal thing added by style check at the top can be ignored and removed. Also 2.0.3 is out.

**Comment by AdamXweb** [comment]
Removed unwanted strings, and bumped version to v2.0.3

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**